<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Style-Type" content="text/css">
  <meta name="generator" content="pandoc">
  
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="/style.css" type="text/css">
          <title>Genetics</title>
</head>
<body>

<h1 id="genetics">Genetics</h1>          <a href="index.html">Back to UWaterloo</a>
<div id="TOC">

<ul>
<li><a href="#mendelian">Mendelian</a></li>
<li><a href="#mitosis----chapter-3">Mitosis -- Chapter 3</a></li>
<li><a href="#changes-in-chromosome-number">Changes in Chromosome Number</a></li>
<li><a href="#linkage-and-recombination">Linkage and Recombination</a></li>
<li><a href="#map-distances">Map distances</a></li>
<li><a href="#dna">DNA</a></li>
<li><a href="#dna-replication">DNA Replication</a></li>
<li><a href="#genetic-code">Genetic Code</a></li>
<li><a href="#transcription">Transcription</a></li>
<li><a href="#translation">Translation</a></li>
<li><a href="#gene-regulation-in-prokaryotes">Gene Regulation in Prokaryotes</a></li>
<li><a href="#lac-operon">Lac Operon</a></li>
<li><a href="#population">Population</a></li>
<li><a href="#restriction-enzymescloning">Restriction Enzymes/Cloning</a></li>
<li><a href="#cancer-genetics">Cancer Genetics</a></li>
</ul>
</div>
<ul>
<li>mendelian inheritance, how it relates to genotype, phenotype and alleles</li>
<li>differences of gene structure between prokaryotes and eukaryotes</li>
<li>Drosophal</li>
<li>when breeding, to figure out genotypes, we want to take pure bred Ps, cross to form F<sub>1</sub>, and cross F<sub>1</sub> to get F<sub>2</sub>, then inspect ratio to determine whether the trait is complete dominance (1:3), codominance (1:2:1), incomplete dominance (1:2:1), two genes interacting (9:3:3:1), complementary genes (9:7)</li>
</ul>
<h4 id="introduction">Introduction</h4>
<p>Mendelian genetics, pedigree analysis is applied for humans</p>
<ul>
<li>
<strong>phenotype</strong> is something we can see &amp; observe (trait); end result rather than genetic representation</li>
<li>
<strong>genotype</strong>: genetic make up; description of the genetic information (what are the alleles that make it up)</li>
<li>
<strong>progeny</strong>: descendants</li>
<li>Mendel's experiments with garden peas: <strong>cross-fertilization/cross-pollination</strong>
</li>
<li>anthers that contain pollen and cross-fertilize using pollen between purple and white (different phenotype) pea flowers<ul><li>cross fertilizing weren't clones!</li></ul>
</li>
<li>
<strong>monohybrid crosses</strong> come from pure breeds: ex. pure yellow pea and pure green pea</li>
<li>breeding pures, they're called F<sub>1</sub> progeny, or first filial (monohybrids since only one trait has evolved)</li>
<li>Discrete units of inheritance are <strong>alleles</strong> of genes, and are instances of a single gene<ul><li>
<em>hair colour</em> is the gene (object), black and brown are alleles (instances)</li></ul>
</li>
<li>diploid organisms means they only ever have two alleles for a given gene as a part of their DNA (more than one allele may exist among a population though)</li>
<li>by convention, <em>genes</em> are written in italics</li>
<li>
<strong>homozygous</strong> means both alleles are the same for a gene vs. <strong>heterozygous</strong>
</li>
<li><strong>polymorphic</strong></li>
<li>
<strong>gametes</strong> are the reproductive cells</li>
<li>
<strong>Mendel's law of segregation</strong>: two alleles for each trait separate during gamete formation and unite at random during fertilization</li>
<li>Peas are wrinkled based on whether they are soluble or not<ul><li>starch active branching enzyme makes the starch insoluble, acting catalytically</li></ul>
</li>
<li>
<strong>Dihybrid crosses</strong> with respect to two genes have different alleles for both genes; come from two pure bred homozygous dominant and recessive parents respectively</li>
<li>
<strong>Mendel's Law of Independent assortment</strong>: during gamete formation, different pairs of alleles segregate independently of each other</li>
<li>by summing genotypic ratios, we get the phenotypic ratios 1:2:1 and 9:3:3:1 for one and two genes respectively</li>
<li>a <strong>test cross</strong> is used to find a genotype from a phenotype</li>
</ul>
<h3 id="mendelian">Mendelian</h3>
<p>Genes can have incomplete dominance or codominanance, resulting in new phenotype ratios</p>
<ul>
<li>some genes might have multiple possible alleles, in which case you can determine a <strong>dominance series</strong> by crossing every combination of homozygotes</li>
<li>
<strong>complementation</strong> between two genes is when the phenotype changes if either of two genes has homozygous recessive (mutations in the same gene result in the same phenotypic offspring, while mutations in different genes results in the new phenotype in F1)</li>
<li>
<strong>epistatic</strong> is control over another gene<ul>
<li>we see three phenotypes: one for both dominant, and two for at least one recessive phenotype (where only one shows through for both recessive)</li>
<li>that is when F2 is homozygous recessive for the epistatic gene, then we see a new phenotype. The ratio is 9:3:4 for <strong>recessive epistasis</strong>
</li>
</ul>
</li>
<li>
<strong>hypothesis testing</strong> let's us use different breeding tests to decide between hypotheses</li>
<li>genotype does not always dictate phenotype, sometimes environment influences it</li>
<li>
<strong>pedigree analysis</strong> allows us to track inheritance of human traits</li>
</ul>
<h3 id="mitosis----chapter-3">Mitosis -- Chapter 3</h3>
<p>Interphase gap -- cells are activitly creating whatever they should be creating in the biological role</p>
<ul>
<li>individual form sister chromatids remain joined at the centromere</li>
<li>metaphase chromosomes are either <strong>metacentric or acrocentric</strong> depending on the location of the centromere</li>
</ul>
<p>Interphase Gap 2</p>
<ul><li>phase before mitosis, cells synthesize proteins for mitosis</li></ul>
<p><strong>Mitosis:</strong> Prophase</p>
<p>Mitosis:</p>
<ul>
<li>Prophase<ul><li>kinetochore is where microtubules of the spindle attach</li></ul>
</li>
<li>Metaphase -- metaphase plane is formed</li>
<li>Anaphase -- sister chromatids move to opposite poles, <strong>disjunction</strong>
</li>
<li>Telophase and cytokinesis<ul>
<li>animal, contractile ring squeezes creating the cleavage furrow to split cells</li>
<li>plant, you have to lay down celulose at the cell plane to divide cells</li>
</ul>
</li>
</ul>
<p>unipartite, bipartite?</p>
<h4 id="meiosis">Meiosis</h4>
<p>Germ cells divide to produce gametes through meiosis.</p>
<p>Germ cells go through Meiosis. First the chromosomes go through duplication.</p>
<ul><li>meiosis I is separation of chromosomes: 2n-&gt;n<ul>
<li>centromere remains entact</li>
<li>homologous pairs will cross and swap genetic material</li>
</ul>
</li></ul>
<p>Prophase I:</p>
<ul>
<li>Leptotene, chromosomes condense</li>
<li>Zygotene, chromosomes seek out homologous partner, so they line up. <strong>Synaptonemal complex</strong> form (protein structure that brings them together like a zipper)<ul><li>
<strong>tetrad</strong>s form from two homologous chromosomes pairs (4 chromatids total)</li></ul>
</li>
<li>Pachytene, <strong>crossing over</strong> is genetic exchange between non-sister chromatids of homologous pair, and get a hybrid between maternal and paternal<ul><li>
<strong>recombination nodules</strong> appear along the synaptonemal complex, facilitating DNA exchange (3-4 sites)</li></ul>
</li>
<li>Diplotene, synaptonemal complex dissolves<ul><li>crossing over gives us independent assortment</li></ul>
</li>
<li>Diakinesis, separation occurs</li>
</ul>
<p>Metaphase I:</p>
<ul><li>microtubules pulls chromosomes</li></ul>
<p>Anaphase I:</p>
<p>Meiosis II:</p>
<ul><li>Anaphase II, centromere breaks</li></ul>
<p>Comparing:</p>
<ul>
<li>Mitosis for somatic cells, meiosis for germ cells (in the sexual cycle)</li>
<li>Mitosis duplicates identitcal cells, Meiosis facilitates cross-over and exchange of genetic material</li>
<li>Mitosis is one stage, Meiosis is two stages of division (since haploid cells are the results)</li>
</ul>
<p><strong>Oogenesis</strong> is egg formation in humans. The egg need lots of cytoplasm with mitochondria</p>
<ul>
<li>start with diploid cells (divided through mitosis)</li>
<li>
<em>is like meiosis but three of the four haploids produced disintegrate in what we call</em> <strong>polar bodies</strong>
</li>
<li>create a <strong>secondary oocyte</strong><ul><li>undergoes another round of meiosis</li></ul>
</li>
<li>store oocytes in ovaries arrested at prophase I diplotene after corssing over, and release mature haploid oocytes after puberty<ul>
<li>ovum and sperm fuse to form the diploid zygote (formation of two gametes)</li>
<li>since oocytes are suspended at prophase I, with microtubules attached to kinetochores, there may be segregation errors as they degrade resulting in a trisomy</li>
</ul>
</li>
</ul>
<p>Looking at Drisophila:</p>
<ul>
<li>
<strong>Hemizygous</strong> for a particular chromosome means you only have one copy of the allele.</li>
<li>
<strong>Nondisjunction</strong> occurs when an error in Meiosis causes X chromosomes in females to not separate uniformly, so two end up together in one egg and nullo-X in the other</li>
<li>
<strong>chromosomal theory of inheritance</strong> is affirmed, and the conclusion is that genes reside on specific chromosomes, genes determine traits</li>
<li>Thomas Hunt Morgan showed that crisscross inheritance occurs in X-linked traits (mother passes to son, father passes to daughter)<ul><li>XXX dies usually since</li></ul>
</li>
</ul>
<h3 id="changes-in-chromosome-number">Changes in Chromosome Number</h3>
<p>Aneuploidy - individual whose chromsome number is not a multiple of <code>n</code></p>
<ul>
<li>Monosomic 2n -1</li>
<li>Trisomic 2n + 1</li>
<li>Tetrasomic 2n + 2 four copies of a given chromosome</li>
</ul>
<p>Autosomal aneuploidy is generally lethal</p>
<ul>
<li>chromosome 1 and 2 are large so almost always lethal</li>
<li>Trisomy 21 is tolerated well since it is a small chromosome</li>
<li>Nondisjunction of hommight happen after cross over has occurred in meiosis I, in which case we get two diplo-21 gametes and two nullo-21</li>
<li>nondisjunction after Meiosis II, we get two normal, one diplo-21, one nullo-21 and thus one trisomy, and two normal, one monosomy</li>
</ul>
<p>Aneuploidy for the X chromosome</p>
<ul><li>
<strong>X-inactivation</strong>, all but one X chromosome is repressed, otherwise females will have all genes expressed twice<ul>
<li>happens randomly per cell</li>
<li>Bar body is the condensed inactive X</li>
<li>tendency to clonally keep the same X off</li>
<li>calico multi-coloured cats are a result of inactivation of random X's, for codominant alleles</li>
</ul>
</li></ul>
<p>Sex Chromosome Aneuploidy</p>
<ul>
<li>XXX 1:1000, normal</li>
<li>XO (one X) 1:2500 Turner's Syndrome</li>
<li>XXY 1:500 Klinefelter Syndrome</li>
<li>XYY</li>
<li>OY: lethal</li>
</ul>
<p>Kleinfelter's results from non-disjunction in female in Meiosis I or Meiosis II</p>
<p>Kleinfelter's results from non-disjunction in males in only Meiosis I.</p>
<p><strong>Aneuploid mosaics</strong> like Turner's Syndrome result from mitotic nondisjunction</p>
<ul>
<li>the centromere doesn't split so we can end up with a trisomic cell and monosomic cell</li>
<li>lagging chromatid at anaphase can be lost and cause monosomic cells</li>
<li>Turner's syndrome only works if you are an aneuploid mosaic</li>
</ul>
<p><strong>Gynandromorph</strong> is a special case where nondisjunction occurs in the first mitosis division.</p>
<p><strong>Euploids</strong> contain only complete sets of chromosomes</p>
<ul>
<li>
<strong>Polyploids</strong> carry three or more complete sets of chromosomes (more gene expression)</li>
<li>
<strong>Monoploids</strong> are organsims which have one complete set of set, and are generally infertile</li>
<li>Note "somic" =&gt; aneuploid, "ploid" =&gt; euploid</li>
</ul>
<p>Whiptail lizards are only females.</p>
<p>Polyploids (like Triploids and Tetraploids) are almost always sterile, since during cell division, chromatids do not segregate correctly</p>
<ul><li>Salmon can segragate correctly since they tag maternal and paternal</li></ul>
<h3 id="linkage-and-recombination">Linkage and Recombination</h3>
<p>If genes are on different chromosomes, they will assort independently</p>
<ul>
<li>but if two genes are close together on a chromosome, they will be <strong>linked</strong> and <strong>recombination</strong> will be rare</li>
<li>
<strong>parental phenotype</strong> are the same exhibited phenotypes as either pure bred parent (in dihybrid crosses)</li>
<li>
<strong>recombinant/nonparental phentotypes</strong> are parts of the parents</li>
<li>mendel's law of independent assortment predicts that in the dihybrid cross, all 4 phenotypes will occur with ratio 1:1:1:1<ul>
<li>if genes are linked, we'll get more than 50% parentals, and vise versa</li>
<li>the more recombinants, the further apart the genes are on a chromosome</li>
</ul>
</li>
<li>|recombinants| ≤ |parental|</li>
</ul>
<p>To keep track of parental allelic combinations, we can represent genotypes:</p>
<ul>
<li>vg b / vg<sup>+</sup> b<sup>+</sup>, rather than vg vg<sup>+</sup> b<sup>+</sup> b to hold more information</li>
<li>take the F1 female and cross with an all recessive male to determine parental : recombinant ratio<ul><li>we choose recessive male when testing with F1 since we don't want their traits to show</li></ul>
</li>
<li>if we only look at the males, then the male parent genotype doesn't matter</li>
</ul>
<p>The Chi Square Test is a statistical test to evaluate hypotheses that two genes assort independently</p>
<ul>
<li>basically to determine how statistically significant an outcome is</li>
<li>apply an null hypothesis of no linkage</li>
</ul>
<ol>
<li>Use the data from an experiment, to determine total number of offspring and classes, and the given distribution</li>
<li>Calculate the expected number (based on 1:1 ratio for null hypothesis).</li>
<li>Χ^2 = ∑ (observed - expected)^2/expected</li>
<li>Determine the degrees of freedom (df) = number of classes - 1</li>
<li>Use the chi-square value and number of degrees of freedom to determine a <code>p value</code> = probability that a deviation from the predicted numbers at least as large as that observed in the experiment will occur by chance</li>
<li>Evaluate the significance of the p value. <strong>0.05 p value</strong> is the boundary between accepting and rejecting null hypothesis<ul>
<li>p=0.05 corresponds to two standard deviations</li>
<li>p value that's high means that there is a higher probability that the result was due to chance</li>
</ul>
</li>
</ol>
<h3 id="map-distances">Map distances</h3>
<p>Recombination was demonstrated using <strong>physical markers</strong> that were visible under the mircoscope. In the example, there was discontinuity and a bend.</p>
<ul>
<li>using <strong>recombination frequency</strong> as the percentage, we can measure it using <strong>mapping units</strong>
</li>
<li>by convention, <code>ry / Ry ; tV / Tv</code> is notation for maternal / paternal and semi colon indicates different chromosomes</li>
<li>recombination frequencies of 50% indicates either not on the same chromosome, or they're really far apart on the same<ul><li>we can discern between the two by looking at each linkage with a gene between them</li></ul>
</li>
</ul>
<p>Two Point Crosses</p>
<ul><li>Limitation is that mapping units are less accurate when genes are far apart, due to high variance</li></ul>
<p>Three Point crosses</p>
<ul>
<li>double crossovers messes up the data, since crosses can occur without changing the gamete class</li>
<li>2^3 = 8 different genotypes</li>
<li>highest two are the parents, always</li>
<li>second two highest are single crossovers in the largest region I, SCOI</li>
<li>third highest two are single crossovers in the smaller region II, SCOII</li>
<li>last two will be double crossovers, one in each region, DCO</li>
<li>Next, for all 3 different pairs of genes, sum all the recombinants (ignoring the third) and multiply by 100 to get m.u.</li>
<li>using m.u. we can determine order</li>
<li>finally, we add the number of DCO <strong>twice</strong> (since two crossovers occur) to the pair of genes with the farthest m.u. to account for the error messing up the map units, since we don't get a recombinant phenotype</li>
<li>pick one parental class, and pick one double crossover, and odd one out will be the one in the middle</li>
</ul>
<p>Double crossovers might not occur because they are rare.</p>
<ul>
<li>another possibility is <strong>interference</strong>, genes that are really close, or close to the centromere, or near the telomeres, are all reasons that double crossovers might not occur ( or pr(I) * pr(II) ≠ pr(DCO) )</li>
<li>
<strong>coefficient of coincidence</strong> is a measure of the amount of interference (0 = no interference)<br>
 - coefficient of coincidence = observed/expected, and interference = 1 - coefficient of coincidence; so 30% interference means 30% less than what we expect</li>
</ul>
<h3 id="dna">DNA</h3>
<ul>
<li>weakly acidic, phosphorous rich, from the nuclei of white blood cells</li>
<li>phosphorous links</li>
<li>thought that proteins were</li>
<li>
<strong>Frederick Griffith</strong> performed experiments with bacteria<ul>
<li>capsule prevents the immune system from stopping bacteria</li>
<li>he showed that they achieved <strong>transformation</strong>, a substance changing genetic characteristics</li>
</ul>
</li>
</ul>
<p>What component is responsible for this phenomenon of transforming cells?</p>
<ul><li>Oswald Avery <strong>transforming principle</strong><ul>
<li>took the heat killed smooth, and treated with different enzymes</li>
<li>protease is an enzyme that chops up all the protien, and yet transformation still occurred so it wasn't protein</li>
<li>RNase chews up RNA, introduced into rough form cells and there was still transformation</li>
<li>DNase destroyed DNA, and there wasn't transformation</li>
</ul>
</li></ul>
<p>Chase and Hershey Experiments with infecting bacterial cells with bacteriophages</p>
<ul><li>bacteriophage injects bacteria with DNA, T2 phage<ul><li>grow inside cells and then are released when the cell bursts</li></ul>
</li></ul>
<p>Structure of DNA: Deoxyribose sugar is a pentagon of carbons</p>
<ul>
<li>5' is connected to 4', not part of pentagon</li>
<li>5' connected to one phosphodiester and 3' connected to another phospherdiester</li>
<li>directed from 5' -&gt; 3'</li>
</ul>
<p>DNA is a double helix, first proposed by James Watson and Francis Crick</p>
<ul>
<li>Rosalind Franklin was a scientist who helped to discover</li>
<li>Purines (Adenine and Guanine) are always connected by a hydrogen bond to a Pyramidine (Thymine or Cytosine)</li>
<li>has multiple forms; B-form and spirals right</li>
<li>Z form is the left form highly active form</li>
<li>A form is dehydrated</li>
</ul>
<h3 id="dna-replication">DNA Replication</h3>
<p>Watson-Crick model of replication is <strong>semiconservative</strong>; one strand is from the parental molecule and other is newly synthesized</p>
<ul>
<li>double helix separates, and complementary bases align opposite to <strong>templates</strong>
</li>
<li>enzymes link sugar-phosphate elements of newly aligned nucleotides</li>
<li>label dna using radiation, centrifuge DNA in Cesium choloride</li>
</ul>
<p><strong>Meselson-Stahl Experiment</strong>: use atomic weights to show how DNA replicates.</p>
<ol>
<li>E coli is grown on <sup>15</sup>N for several generations (so that the DNA itself is atomically heavier)</li>
<li>DNA is extracted and density is analyzed through centrifugation (visible bands determine atomic weight)</li>
<li>Cells are transferred to a lighter medium <sup>14</sup>N for one generation.</li>
<li>DNA is analyzed again, but this time it is precisely 14N and 15N on the gradient. "Hybrid"</li>
<li>More generations, less and less hybrid and more 14</li>
</ol>
<p>This proves the semiconservative model.</p>
<p>Two steps; initiations and elongation</p>
<ul>
<li>origin of replication, rich in AT sequencing</li>
<li>AT have two hydrogen bonds, so it's easy to break apart<ul><li>heating breaks hydrogren bonds</li></ul>
</li>
<li>initiator protein and DNA helicase are at the replication fork to open up the replication bubble</li>
</ul>
<p><strong>Elongation</strong>: DNA polymerase must have</p>
<ul>
<li>DNA primase puts down RNA primer</li>
<li>RNA primer is from 5' to 3'</li>
<li><strong>primer</strong></li>
</ul>
<p>One kind of E coli DNA polymerase, III, type of enzyme that is used to extend DNA using RNA primers</p>
<ul>
<li>leading strand</li>
<li>template is always 3' to 5'</li>
</ul>
<p>DNA polymerase I removes RNA primer and fills in gaps, still from 5' to 3'</p>
<p>DNA <strong>ligase</strong> joins leading and lagging strands by the phosphodieseter</p>
<p>E coli DNA Polymerase adds nucleotides to the end of a DNA stand</p>
<ul>
<li>can only add nucleotides in the 5' to 3' direction</li>
<li>needs a free 3' hydroxide to polymerize</li>
</ul>
<p>Three <strong>activities</strong> for our two types of E. coli DNA polymerase</p>
<ol>
<li>5' to 3' polymerase activity</li>
<li>3' to 5' exonuclease activity (chewing it off) and fixing mistakes</li>
<li>5' to 3' exonuclease primer degradation</li>
</ol>
<p>Problem with circular chromosomes: <strong>positive supercoils</strong>, tension opened by <strong>topoisomerases</strong></p>
<ul><li>nicks both DNA strnads and separate the two daughter molecules</li></ul>
<p>Eukaryotes have <strong>linear chromosomes</strong></p>
<ul>
<li>
<strong>telomeres</strong> are at the end of each chromosome</li>
<li>telomeres are smaller and smaller after each replication<ul><li>repeated units of TTAGGG</li></ul>
</li>
</ul>
<h4 id="dna-replicaiton-process">DNA Replicaiton Process</h4>
<ol>
<li>Two Helicases, one at each replication fork of the DNA</li>
<li>Primase make exactly two RNA primers</li>
<li>DNA Polymerase III extends out, <em>copying DNA to mRNA</em>
</li>
<li>Repeat 2 &amp; 3</li>
<li>DNA Polymerase I fixes RNA primers, leaving a nick at the end</li>
<li>Ligase mends the nicks</li>
</ol>
<h3 id="genetic-code">Genetic Code</h3>
<p>DNA acts as a template for RNA</p>
<ul>
<li>
<strong>transcription</strong> is ATCG to AUCG, and the messenger RNA can go through <strong>translation</strong> to protein</li>
<li>grouping into triplets, we get 4^3 = 64 <strong>codons</strong>
</li>
<li>AUG =&gt; Met or methymine is the start codon</li>
<li>UAG, UAA, UGA are the stop codons, translation stops</li>
<li>gene's nucleotide is colinear with the encoded <strong>polypeptide</strong>
</li>
<li>from DNA, what is the linear order of the amino acids which determines how the protein folds</li>
<li>
<strong>reading frame</strong>, the partitioning of groups of three nucleotides</li>
<li>
<strong>frameshift mutation</strong> might make the reading frame not make sense</li>
<li>
<strong>intragenic suppression</strong> restoration of gene function by one mutation cancelling another in the same gene<ul>
<li>sometimes the mutated section doesn't affect the resulting protein</li>
<li>multiples of three are ok, or deletions and additions might cancel out</li>
</ul>
</li>
<li>genetic code is almost universal, but there are exceptions</li>
</ul>
<p>Researchers used radioactive labelling (C14 instead of C12) which makes no difference in DNA structure</p>
<ul>
<li>mRNA is very unstable</li>
<li>code was cracked, by filtering DNA (some experiment)</li>
<li>RNA-likes trand is 5' to 3'</li>
<li>Template strand is 3' to 5'</li>
<li>mRNA is made from template strand</li>
<li>
<strong>polypeptide</strong> is a chain of amino acids<ul><li>amino and carboxy at the terminus</li></ul>
</li>
</ul>
<h3 id="transcription">Transcription</h3>
<p>The process by which the polymerization of ribonucleotides is guided by complementary base pairing (template?) to produce and <strong>RNA transcript</strong> of a gene</p>
<ul>
<li>nucleotides will be added 5' to 3' direction</li>
<li>Uracil replaces Thymine</li>
<li>two types of termination<ul>
<li>
<strong>Rho dependent</strong>: protein/factor is binding to the RNA</li>
<li>Rho independent</li>
</ul>
</li>
<li>eukaryotic transcription needs a promoter and includes an enhancer most of the time</li>
<li>AAUAA, poly A tail?</li>
<li>assembly of Pro and Euk, start at <strong>shine dalgarno</strong>
</li>
</ul>
<p>Terminator signal is encoded in the gene</p>
<ul><li>
<strong>rho-dependent</strong> or <strong>rho-independent</strong> are two types of termination, dependent on the rho factor</li></ul>
<h3 id="translation">Translation</h3>
<p>The process in which the genetic code carried by mRNA directs the synthesis of proteins from amino acids</p>
<ul><li>transfer RNAs (tRNA) short single stranded RNA, each carries one pariticular amino acid</li></ul>
<p>Mutations: silent, missense (wrong amino acid), nonsense (cause a stop, truncation), frameshift, or in promoter</p>
<ul><li>prokaryotic promoter -10 TATAAT and -35 concensus (concensus sequence?)</li></ul>
<h3 id="gene-regulation-in-prokaryotes">Gene Regulation in Prokaryotes</h3>
<p>DNA regulation determines which genes are transcribed and translated (think immune cell vs muscle cells have same DNA but different funcitons)</p>
<ul>
<li>gene is a sequence of DNA (RNA polymerase needs to attach somewhere), we call <strong>promoter</strong>
</li>
<li>each gene has a promoter</li>
<li>for prokaryotes, multiple genes can be associated with one promoter (some sequence of DNA)</li>
<li>an <strong>operon</strong> is a sequence of DNA that includes the regulatory DNA sequence, promoter, and its associated genes</li>
<li>what can enhance and inhibit this process?</li>
<li>
<strong>repressor</strong> attaches to a sequence after promoter that represses transcription, regulating gene expression</li>
<li>
<strong>activator</strong> binds for more transcription</li>
<li>
<strong>inducers</strong> turn on activation of the activator</li>
<li>
<strong>lac operon</strong> codes for genes involved in the metabolism of lactose (for E. coli)<ul><li>has promoter, operator, lacZ, lacY, lacA and is preceeded by a <strong>CAP site</strong> (Catabolite ACtivator Protein) binds as an activator</li></ul>
</li>
<li>if there's no lactose, lac repressor is bound to operator</li>
<li>if lactose is present, we have <strong>allolactose</strong> that acts as an inducer</li>
<li>if present, it binds to repressor so it doesn't bind as well, and we can start metabolizing lactose</li>
</ul>
<p>Glucose is another sugar preferred by the bacteria over lactose</p>
<ul>
<li>no glucose and lactose, we'll have an activator resulting in lots of transcription</li>
<li>high glucose -&gt; low CAMP, so less lactose</li>
</ul>
<p>I<sup>-</sup> mutation that signals repressor, cannot bind to the operator</p>
<ul>
<li>I<sup>s</sup>
</li>
<li>p- mutation, RNA cannot associate with it</li>
<li>O<sup>c</sup> repressor protein can't associate with operator</li>
<li>Z- mutation, cannot create allolactose</li>
</ul>
<h3 id="lac-operon">Lac Operon</h3>
<ul>
<li>in the absense of lactose, we don't want to be making proteins if we don't need them, the repressor is going to bind to the operator</li>
<li>this is going to stop transcription</li>
<li>normally it binds to promoter, but without lac I, it binds to operator</li>
<li>repressor, four monomers come together to create a tetramer that binds to the operator on the lac operon</li>
<li>
<strong>allolactose</strong> binds</li>
<li>three genes: LacZ,Y,A, <strong>polycistronic message</strong>, all at once</li>
<li>promoter, followed by operator, then lacZ, lacY, lacA</li>
<li>lacZ encodes Β-galactosidase<ul>
<li>turns lactose into glucose and galactose</li>
<li>alternatively turns lactose into allolactose</li>
</ul>
</li>
<li>lacY encodes permease, transporting lactose and allows it into the inner membrane</li>
<li>lacA doesn't matter too much</li>
<li>repression of lacZ, lacY and lacA are all linked, leading to the idea that they are transcribed as aprt of the same <strong>polycistronic mRNA</strong>
</li>
<li>lacI is upstream part of the repressor gene, and binds to some other part</li>
<li>DNA binding domain</li>
<li>Inducer binding domain, allolactose changes the confirmation, so that this repressor protein cannot bind to the DNA operator</li>
<li>in gene I<sup>-</sup>, a mutation in our repressor with no repressor, <strong>lac operon</strong> is expressed ALL THE TIME<ul><li>this is <strong>constitutive expression</strong>
</li></ul>
</li>
</ul>
<h3 id="population">Population</h3>
<p>Hardy Weinberg Equalibrium</p>
<ul>
<li>baseline estimate for equalibrium, caluclated to show that allele frequency doesn't change</li>
<li>p^2 + 2pq + q^2 = 1 and p + q = 1 for two allele calculations</li>
<li>run through Chi Square test</li>
</ul>
<h3 id="restriction-enzymescloning">Restriction Enzymes/Cloning</h3>
<p>Restriction REN</p>
<ul>
<li>Digest</li>
<li>plasmid is what you're cloning onto</li>
<li>phosphodiester backbone, giving us a plasmid</li>
<li>LB AMP selects for e coli containing a plasmid</li>
</ul>
<h3 id="cancer-genetics">Cancer Genetics</h3>
<p>Genetic disorder involving mutations in cells</p>
<ul>
<li>
<strong>oncogenesis</strong> is the formation of a cancer, first causing tumors if loss of cell cycle control occurs, and cancer is caused if cells undergo further changes, disrupting other tissues</li>
<li>Carcinoma (skin), Sarcoma (connective tissue), Leukemia (blood), Lymphoma (immune system)</li>
<li>cancer arises when cell division no longer function properly</li>
<li>tumor-suppressor genes, involved in cell cycle control, p53 and RB<ul>
<li>p53 is the gatekeeper, it regulates G1 to S checkpoint</li>
<li>if <strong>cyclin-dependent kinase</strong> (CDK) is still active, then cell commits to S phase because it doesn't think there's DNA damage</li>
<li>p53 induces expression of p21, activating CDK which stops cell cycle before S phase</li>
</ul>
</li>
<li>
<strong>Oncogenes</strong> are mutant alleles that act dominantly to stimulate cell proliferation<ul><li>proto-oncogenes</li></ul>
</li>
</ul>
    <div id="footer">
      Notes by <a href="https://github.com/kevintpeng">Kevin Peng</a>, Google intern.<br>
      Formerly Riot Games, Bloomberg, Apple &amp; Shopify.<br>
      Connect with me on <a href="https://www.linkedin.com/in/kevintpeng/">LinkedIn</a>.
    </div>
</body>
</html>
